scripts/check_frontend_test_coverage.py

```diff
@@ -0,0 +1,217 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Check for decrease in coverage from 100% of frontend files."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import os
+import shutil
+import subprocess
+import sys
+
+import python_utils
+
+LCOV_FILE_PATH = '../karma_coverage_reports/lcov.info'
+DEV_LCOV_FILE_PATH = './tmp/dev-lcov.info'
+PR_LCOV_FILE_PATH = './tmp/pr-lcov.info'
+
+
+def change_git_branch(branch):
+    """Changes git branch.
+
+    Args:
+      branch: string. Name of the branch to be changed.
+    """
+    subprocess.check_call(['git', 'checkout', branch])
+
+    if branch == 'develop':
+        # This line is executed because CircleCI keeps the commits from PR
+        # branch when changing branchs. Ref ->
+        # (https://discuss.circleci.com/t/changing-git-branches-causes-
+        # commits-from-one-branch-to-carry-over-to-the-next/13562)
+        subprocess.check_call([
+            'git', 'reset', '--hard', 'origin/develop'])
+
+
+def run_frontend_tests_script():
+    """Run the frontend tests script using subprocess."""
+    subprocess.check_call([
+        'python', '-m', 'scripts.run_frontend_tests'])
+
+
+def create_tmp_folder():
+    """Creates a temporary folder."""
+    if not os.path.exists('./tmp'):
+        os.mkdir('./tmp')
+
+
+def delete_tmp_folder():
+    """Delete the temporary folder."""
+    if os.path.exists('./tmp'):
+        shutil.rmtree('./tmp')
+
+
+def filter_lines(line):
+    """Check if the line has file path (SF) or total lines (LF) or covered
+        lines (LH) of the test.
```

Comment:
Deindent this line by 4.

Reply:
I can't indent it by 4 because pylint throws an error that code should be indented by 2, and blocks the push.

Reply:
Everywhere else in the codebase, code is indented by multiples of 4, so it seems strange that pylint is saying "indent by 2". Could you investigate? /cc @oppia/dev-workflow-team as well.

Also, please note that my comment was to **deindent** the line by 4, i.e. move it 4 spaces to the left so that it's flush with the line above. See other docstrings in the codebase for examples.

Reply:
Can you please give me a example of a file that follows the indentation that you mention? I'm searching it on the project but I think I didn't get it right.

Reply:
Pretty much any file that has a multi-line initial docstring. Like this one: https://github.com/oppia/oppia/blob/develop/core/domain/activity_services.py#L102

Reply:
Thanks, now I get it. Done!